my_library/models/library_book.py

```diff
@@ -1,15 +1,111 @@
-from odoo import models, fields
+import dot_parser
+from datetime import timedelta
+from odoo.addons import decimal_precision as dp
+from odoo import api, models, fields
+from odoo.exceptions import ValidationError
 
 
 class LibraryBook(models.Model):
     _name = 'library.book'
+    _description = 'Library Book'
+
+    _order = 'date_release desc, name'
+    _rec_name = 'short_name'
 
     name = fields.Char(string='Title', required=True)
+    short_name = fields.Char(string='Short Title', required=True)
+    notes = fields.Text(string='Internal Notes')
+    state = fields.Selection(
+        selection=[('draft', 'Not Available'),
```

Comment:
# const.py
STATE_DRAFT = 'draft'
STATE_LOST = 'lost'
...

STATE_LIST = [
    (STATE_DRAFT, 'Not Available'),
    (STATE_LOST, 'Lost'),
    ...
]

# library_book.py
from . import const
state = fields.Selection(selection=const.STATE_LIST, default=const.STATE_DRAFT, string='State')